object-construction-checker/src/main/java/org/checkerframework/checker/objectconstruction/framework/FrameworkSupport.java

```diff
@@ -0,0 +1,31 @@
+package org.checkerframework.checker.objectconstruction.framework;
+
+import com.sun.source.tree.NewClassTree;
+import org.checkerframework.framework.type.AnnotatedTypeMirror;
+
+public interface FrameworkSupport {
+
+  /**
+   * For a toBuilder routine, we know that the returned Builder effectively has had all the required
+   * setters invoked. Add a CalledMethods annotation capturing this fact.
+   *
+   * @param t
+   */
+  public void handleToBulder(AnnotatedTypeMirror.AnnotatedExecutableType t);
```

Comment:
Typo: it should be `handleToBuilder`

Reply:
Also, we should clarify that this method is invoked whenever `t` has the _name_ `toBuilder`.  The `FrameworkSupport` class is still responsible for figuring out if it's the relevant `toBuilder` method